comment/comment_revision_repository_blackbox_test.go

```diff
@@ -0,0 +1,128 @@
+package comment_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/almighty/almighty-core/account"
+	"github.com/almighty/almighty-core/comment"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/almighty/almighty-core/gormsupport/cleaner"
+	"github.com/almighty/almighty-core/migration"
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/rendering"
+	"github.com/almighty/almighty-core/resource"
+	testsupport "github.com/almighty/almighty-core/test"
+	"github.com/almighty/almighty-core/workitem"
+
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestRunCommentRevisionRepositoryBlackBoxTest(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &revisionRepositoryBlackBoxTest{DBTestSuite: gormsupport.NewDBTestSuite("../config.yaml")})
+}
+
+type revisionRepositoryBlackBoxTest struct {
+	gormsupport.DBTestSuite
+	repository         comment.Repository
+	revisionRepository comment.RevisionRepository
+	clean              func()
+	testIdentity1      account.Identity
+	testIdentity2      account.Identity
+	testIdentity3      account.Identity
+}
+
+// SetupSuite overrides the DBTestSuite's function but calls it before doing anything else
+// The SetupSuite method will run before the tests in the suite are run.
+// It sets up a database connection for all the tests in this suite without polluting global space.
+func (s *revisionRepositoryBlackBoxTest) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+	// Make sure the database is populated with the correct types (e.g. bug etc.)
+	if _, c := os.LookupEnv(resource.Database); c != false {
+		if err := models.Transactional(s.DB, func(tx *gorm.DB) error {
+			return migration.PopulateCommonTypes(context.Background(), tx, workitem.NewWorkItemTypeRepository(tx))
+		}); err != nil {
+			panic(err.Error())
+		}
+	}
+}
+
+func (s *revisionRepositoryBlackBoxTest) SetupTest() {
+	s.repository = comment.NewRepository(s.DB)
+	s.revisionRepository = comment.NewRevisionRepository(s.DB)
+	s.clean = cleaner.DeleteCreatedEntities(s.DB)
+	testIdentity1, err := testsupport.CreateTestIdentity(s.DB, "jdoe1", "test")
+	require.Nil(s.T(), err)
+	s.testIdentity1 = testIdentity1
+	testIdentity2, err := testsupport.CreateTestIdentity(s.DB, "jdoe2", "test")
+	require.Nil(s.T(), err)
+	s.testIdentity2 = testIdentity2
+	testIdentity3, err := testsupport.CreateTestIdentity(s.DB, "jdoe3", "test")
+	require.Nil(s.T(), err)
+	s.testIdentity3 = testIdentity3
+}
+
+func (s *revisionRepositoryBlackBoxTest) TearDownTest() {
+	//s.clean()
```

Comment:
ha yes, good point. I removed it during my last tests to verify the state of the data in the DB. Let me remove the comment.